conf/messages.en

```diff
@@ -364,3 +364,26 @@ agentTelephoneNumber.checkYourAnswersLabel = What is the agency’s telephone nu
 agentTelephoneNumber.hint = We will use this number to contact you about this trust’s registration. For international numbers include the country code
 agentTelephoneNumber.error.required = Enter the agency’s telephone number
 agentTelephoneNumber.error.invalid.characters = Enter a telephone number, like 01632 960 001, 07700 900 982 or +44 0808 157 0192
+
+assetInterruptPage.title = Information you need to add assets
+assetInterruptPage.heading = Information you need to add assets
+assetInterruptPage.subheading1 = Money
+assetInterruptPage.paragraph1 = You must give the total amount of money in the trust. If more than one amount has been added to the trust, add them together.
+assetInterruptPage.subheading2 = Business
+assetInterruptPage.paragraph2 = For a business named in the trust deed, you need:
+assetInterruptPage.bullet1 = business name
+assetInterruptPage.bullet2 = the business’s Unique Taxpayer Reference (UTR), if you don’t have this you will need to enter the business’s address
+assetInterruptPage.bullet3 = current value
+assetInterruptPage.subheading3 = Shares
+assetInterruptPage.paragraph3 = For shares named in the trust deed, you need:
+assetInterruptPage.bullet4 = share company name
+assetInterruptPage.bullet5 = share class
+assetInterruptPage.bullet6 = number of shares in the trust and total current value
+assetInterruptPage.paragraph4 = You will also need to choose whether the shares are listed on a stock exchange (quoted shares) or not.
+assetInterruptPage.subheading4 = Property or land
+assetInterruptPage.paragraph5 = For property or land in the trust deed, you need:
+assetInterruptPage.bullet7 = description of the property or land, or address if available
+assetInterruptPage.bullet8 = estimated total value of the property or land
+assetInterruptPage.bullet9 = estimated value of the property or land if the trust only owns a proportion
+assetInterruptPage.paragraph6 = The current value of the property or land is needed, not the value when it first entered the trust. This can be an estimate.
+assetInterruptPage.continue = Continue
```

Comment:
We now have 2 occurrences of a Continue button in messages, can this be made generic?

Reply:
complete